src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static int getRandomLength() {
+        Random random = new Random();
+        return random.nextInt(9) + 1;
+    }
+
+    public static Figure getRandomFigure() {
+        Random random = new Random();
```

Comment:
Take out Random in the class field, then there will be no need to create separately in each method.